### PR TITLE
Adding --noColor argument.

### LINKED ISSUE #40: How to dismiss the color
I really do not want the highlight my replacement, how to dismiss the color???

### PR DESCRIPTION
**Pull request by pose**
Fixes #40: How to dismiss the color

### CODE REVIEW

File: bin/shared-options.js [hunk 1]
```diff
@@ -74,5 +74,9 @@ module.exports = {
         flag: true,
         help: "asynchronously read/write files in directory (faster)",
         hidden: true
+    },
+    noColor: {
```
Comment (harthur) [inline bin/shared-options.js:78]: The other options have dashes if they're two words. Add `full: "no-color"` so that it can be used as `--no-color`

Comment (pose) [inline bin/shared-options.js:78]: I'm a bit confused, this is what I see on my terminal:
screen shot 2014-09-08 at 12 19 28
I can change it to `full: no-color`, I just want to make sure I'm consistent with `ignoreCase` and `fileColor` shall I change those two too?

Thanks!

Comment (pose) [inline bin/shared-options.js:78]: @harthur sorry to insist on this. Just let me know if I should go with `full: 'no-color'` or not. Thanks!

Comment (harthur) [inline bin/shared-options.js:78]: Ah, you're right. Didn't see those two, that's fine then.

**Comment by pose** [comment]
Great, thanks!

**Pull request merged**